app/main/views/audits.py

```diff
@@ -1,15 +1,17 @@
-from flask import jsonify, abort, request, current_app
 from datetime import datetime, timedelta
-from ...models import AuditEvent
-from sqlalchemy import asc, desc, Date, cast
+from flask import jsonify, abort, request, current_app
+from sqlalchemy import asc, desc
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy.sql.expression import true, false
-from ...utils import pagination_links, get_valid_page_or_1
-from .. import main
-from ... import db, models
+
 from dmapiclient.audit import AuditTypes
 from dmutils.config import convert_to_boolean
 from dmutils.formats import DATE_FORMAT
+
+from .. import main
+from ... import db, models
+from ...utils import pagination_links, get_valid_page_or_1
```

Comment:
Good shout on the duplicates.

The ordering is my thing actually. I've only done it where I've been changing an import IIRC. I mainly do it so I can read them better and see what's going on. Obviously didn't help in this case.